README.md

```diff
@@ -46,10 +46,6 @@ You can set up a complete cabot stack easily using docker-compose.
 
 - Copy your cabot config to conf/production.conf
 
-- Run the migrations
-
-> `docker-compose run --rm web python /manage.py migrate`
-
```

Comment:
This isn't needed in the docker-compose instructions as the docker-compose command runs migrations by default